src/components/rubric.js

```
@@ -403,9 +403,18 @@ export class Rubric extends composeMixins(
 	}
 
 	_checkHasWorkingCopy() {
-		if (!this.assessmentHref) return false;
-		const assessmentUrl = new URL(this.assessmentHref);
-		return !!assessmentUrl.searchParams.get('workingCopyId');
+		if (!this.assessmentHref) {
+			return false;
+		}
+
+		try {
+			const paramString = this.assessmentHref.split('?')[1];
+			const params = new URLSearchParams(paramString);
+
+			return !!params.get('workingCopyId');
+		} catch (_) {
+			return false;
+		}
```

Ignore any error and just return `false`. This will also work with relative URL's which are used in testing. Tested manually using demo pages.